src/main/java/org/apache/maven/report/projectinfo/dependencies/renderer/DependenciesRenderer.java

```diff
@@ -591,16 +595,75 @@ private void renderSectionDependencyFileDetails() {
                         fileLength = "-";
                     }
 
-                    tableRow(hasSealed, new String[] {
-                        name,
-                        fileLength,
-                        String.valueOf(jarDetails.getNumEntries()),
-                        String.valueOf(jarDetails.getNumClasses()),
-                        String.valueOf(jarDetails.getNumPackages()),
-                        jdkRevisionCellValue,
-                        debugInformationCellValue,
-                        sealedCellValue
-                    });
+                    if (jarDetails.isMultiRelease()) {
+                        String htmlBullet = "&#160;&#160;&#160;&#x2022; ";
+                        String rootTag = htmlBullet + getI18nString("file.details.multirelease.root");
+                        String versionedTag = htmlBullet + getI18nString("file.details.multirelease.versioned");
+
+                        // general jar information row
+                        tableRow(hasSealed, new String[] {
+                            name,
+                            fileLength,
+                            String.valueOf(jarDetails.getNumEntries()),
+                            "",
+                            "",
```

Comment:
Does it make sense to omit num of classes/packages here? What do you think? Hard to tell...

Reply:
I prefer to omit them, since the getNumClasses() and getNumPackages() are already asociated to the root content. In addition, we cannot sum over the root class/package names with the versioned class/package names because some of them will be in more than one place, so putting a "total" in the first row seems somewhat inaccurate.